src/Enums/Fees.php

```diff
@@ -22,5 +22,5 @@ class Fees
 
     public const USERNAME_RESIGNATION = '2500000000';
```

Comment:
all the fees here are unused now I assume? 🤔 since it's all based on `gas_price` and `gas_limit` to determine the actual fee